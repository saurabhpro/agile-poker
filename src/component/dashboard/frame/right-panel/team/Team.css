.team {
    border-color: black;
    border-width: 20px;
 
    font-family: helvetica;
}

.team>h2 {
    border: 2em;
    border-bottom: 5px;
}