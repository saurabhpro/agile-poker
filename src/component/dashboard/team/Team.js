import React, { Component } from 'react';

export default class Team extends Component {
  render() {
    return (
      <div>
        <li>Ram</li>
        <li>Shyam</li>
        <li>ok</li>
      </div>
    );
  }
}
