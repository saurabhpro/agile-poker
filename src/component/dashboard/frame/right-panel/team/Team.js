import React, { useState, useEffect } from 'react';

import TeamMember from './TeamMember';

import './Team.css';
import { getCurrentlyLoggedInUserDetails } from '../../../../utils/firebaseDb';

const Team = ({ className }) => {
  const [teamName, setTeamName] = useState();

  useEffect(() => {
    const getTeam = () =>
      getCurrentlyLoggedInUserDetails().then((usr) =>
        setTeamName(usr.team.toUpperCase()),
      );

    if (!teamName) {
      getTeam();
    }
  }, [teamName]);

  return (
    <div>
      {teamName && (
        <div className="team">
          <div className="team__title">
            <h5
              className={className}
            >{`${teamName.toUpperCase()}`}</h5>
          </div>

          <TeamMember team={teamName} />
        </div>
      )}
    </div>
  );
};

export default Team;
