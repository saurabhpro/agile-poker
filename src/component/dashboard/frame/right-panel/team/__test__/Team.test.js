import React from 'react';
import { shallow } from 'enzyme';
import Team from '../Team';

let component;

describe('result from poker card panel', () => {
  beforeAll(() => {
    component = shallow(<Team className="rightCard" />);
    //console.debug("This was prited",component.debug());
  });

  it('matches the rendered snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render one <TeamMember>', () => {
    expect(component.find('TeamMember')).toHaveLength(0);
  });
});
