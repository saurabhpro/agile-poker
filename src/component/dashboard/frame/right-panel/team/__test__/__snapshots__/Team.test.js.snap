// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`result from poker card panel matches the rendered snapshot 1`] = `<div />`;
